content/en/blog/2022/OpenTelemetry-In-Practice-August.md

```diff
@@ -0,0 +1,55 @@
+---
+title: OpenTelemetry in Practice - Kubernetes & the Collector
+linkTitle: OTel in Practice
+date: 2022-08-11
+author: Rynn Mancuso
+draft: true
+# TODO: remove this line once your post is ready to be published
+# TODO: run the following command before submitting
+#   npx prettier --write content/en/blog/2022/OpenTelemetry-In-Practice-August.md
+---
+
+## About the OpenTelemetry in Practice Series
+
+Welcome to the OpenTelemetry in Practice series! This is a new experiment by
+some OpenTelemetry contributors in the End User Working Group.
+
+We’re aiming to:
+
+- address practical problems that commonly stop development teams from
+  succeeding with OpenTelemetry
+- build stronger connections with developers focused on specific languages
+- improve the experience of implementing OpenTelemetry in production
```

Comment:
Capitalize the first letter of each sentence

Reply:
Isn't each bullet point a continuation of the sentence that starts with "We're aiming to"? For example, "We're aiming to address..", "We're aiming to build...", etc.?

Reply:
See, https://developers.google.com/style/capitalization